sysutils/dcfldd/Portfile

```diff
@@ -1,29 +1,31 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                dcfldd
-version             1.3.4-1
-revision            1
-checksums           rmd160  3a6fba197686da8f1890d21f75b6bc2ae18592a7 \
-                    sha256  f5143a184da56fd5ac729d6d8cbcf9f5da8e1cf4604aa9fb97c59553b7e6d5f8 \
-                    size    163141
+github.setup        ressurecting-open-source-projects dcfldd 1.9.1 v
+github.tarball_from releases
 
+name                dcfldd
```

Comment:
"name" is deduced from the "github.setup" line

Reply:
Macports complained when I tried without "name".

Reply:
Hmm. I tried it again and had no problems. My bad, must have been a mistake here. I will make the changes and test them again right now.

Reply:
Removed. It worked fine now without it.